internal/service/cluster/resource_cluster_test.go

```diff
@@ -1451,9 +1451,9 @@ func TestAccClusterRSCluster_withDefaultBiConnectorAndAdvancedConfiguration_main
 				ConfigPlanChecks: resource.ConfigPlanChecks{
 					PostApplyPreRefresh: []plancheck.PlanCheck{
 						acc.DebugPlan(),
```

Comment:
do we still need this as well?

Reply:
thanks for noting, not necessary as ExpectEmptyPlan() gives details of the plan if it is non empty.